src/4.3_to_5.0/__snapshots__/migrate_43_to_50.test.ts.snap

```diff
@@ -2247,12 +2247,8 @@ exports[`migrate_43_to_50 returns a valid ActiveUI5 /ui folder on a small input
                     "content": "{"name":"hidden grand totals"}",
                     "isDirectory": true,
                     "lastEditor": "admin",
-                    "owners": [
-                      "admin",
```

Comment:
Here too, it would be nice to not lose this, in order to ensure that the migration script does not erase bookmark permissions.